benches/alloc.rs

```diff
@@ -5,9 +5,8 @@ use mmtk::util::test_util::fixtures::*;
 use mmtk::AllocationSemantics;
 
 pub fn bench(c: &mut Criterion) {
-    // Disable GC so we won't trigger GC
-    let mut fixture = MutatorFixture::create_with_heapsize(1 << 30);
-    memory_manager::disable_collection(fixture.mmtk());
+    // Setting a larger heap so we won't trigger GC, but we should disable GC if we can
+    let mut fixture = MutatorFixture::create_with_heapsize(usize::MAX);
```

Comment:
Given that our current GC trigger does align-up for the heap size, like this:

```rs
// GCTrigger::new()
                GCTriggerSelector::FixedHeapSize(size) => Box::new(FixedHeapSizeTrigger {
                    total_pages: conversions::bytes_to_pages_up(size),
                }),
```

`usize::MAX` will be aligned to 0.  So we can't use `usize::MAX` now.  Maybe we can choose a slightly smaller value, such as 1TB or 256TB

Because it is based on `MockVM`, we can mock `is_collection_disabled`, too.